scripts/pre_commit_linter.py

```diff
@@ -2442,7 +2442,7 @@ def _check_constants_declaration(self):
             for filepath in ts_files_to_check:
                 # Check that the constants are declared only in a *.constants.ts
```

Comment:
Update this comment to reflect that the constants should be in `*.constants.ajs.ts`

Reply:
Done!